week-5/A-dom-manipulation/exercise.js

```diff
@@ -5,7 +5,15 @@ Task 1
 Write JavaScript below that changes the background colour of the page when the 'Change colour' button is clicked.
 */
 
+var btn = document.getElementById("bgrChangeBtn");
 
+btn.addEventListener("click", changeBackground);
+
+function changeBackground() {
+  var randomColour = ["blue", "purple", "yellow", "green", "white"];
```

Comment:
Nitpick: this variable name implies that it itself is a single colour. If you renamed it to something like `colors` then that frees up the variable below to be named `randomColor`.

Incidentally, in my experience it's usually better to use the American spelling of colour in code. That's how the computer spells it (see `backgroundColor`), so it can become confusing switching between the 2 😄

Reply:
Hey Ali!

This file has also been updated. Please let me know if you have any additional comments.